presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -142,54 +147,68 @@
 
     @Inject
     public GlueHiveMetastore(HdfsEnvironment hdfsEnvironment, GlueHiveMetastoreConfig glueConfig)
-    {
-        this(hdfsEnvironment, glueConfig, createAsyncGlueClient(glueConfig));
-    }
-
-    public GlueHiveMetastore(HdfsEnvironment hdfsEnvironment, GlueHiveMetastoreConfig glueConfig, AWSGlueAsync glueClient)
     {
         this.hdfsEnvironment = requireNonNull(hdfsEnvironment, "hdfsEnvironment is null");
         this.hdfsContext = new HdfsContext(new ConnectorIdentity(DEFAULT_METASTORE_USER, Optional.empty(), Optional.empty()));
-        this.glueClient = requireNonNull(glueClient, "glueClient is null");
+        this.glueClient = requireNonNull(createAsyncGlueClient(glueConfig, hdfsEnvironment.getConfiguration(hdfsContext, null)), "glueClient is null");
```

Comment:
Instead of passing the `Path` as null can we pass it as `new Path("file:///tmp")` so the null handling on `HiveHdfsConfiguration` and `HdfsEnvironment` can be avoided.